src/renderer/coremods/commands/commands.ts

```diff
@@ -0,0 +1,424 @@
+                type: plugins.plugins.get(addonId)
+                  ? Messages.REPLUGGED_PLUGIN
+                  : Messages.REPLUGGED_THEME,
+                name:
+                  plugins.plugins.get(addonId)?.manifest?.name ??
+                  themes.themes.get(addonId)?.manifest?.name,
+              }),
+            },
+          ],
+        };
+      } catch (err) {
+        return {
+          send: false,
+          embeds: [
+            {
+              type: "rich",
+              color: 0xdd2d2d,
+              title: Messages.REPLUGGED_COMMAND_ERROR_GENERIC,
+              description: err as string,
+            },
+          ],
+        };
+      }
+    },
+  });
+  injector.utils.registerSlashCommand({
+    name: Messages.REPLUGGED_COMMAND_LIST_NAME,
+    description: Messages.REPLUGGED_COMMAND_LIST_DESC,
+    options: [
+      {
+        name: "send",
+        displayName: Messages.REPLUGGED_COMMAND_LIST_OPTION_SEND_NAME,
+        description: Messages.REPLUGGED_COMMAND_LIST_OPTION_SEND_DESC,
+        type: ApplicationCommandOptionType.Boolean,
+        required: false,
+      },
+      {
+        name: "type",
+        displayName: Messages.REPLUGGED_COMMAND_LIST_OPTION_TYPE_NAME,
+        description: Messages.REPLUGGED_COMMAND_LIST_OPTION_TYPE_DESC,
+        type: ApplicationCommandOptionType.String,
+        required: true,
+        choices: [
+          {
+            name: Messages.REPLUGGED_THEME,
+            displayName: `List themes`, // TODO: i18n
+            value: "theme",
+          },
+          {
+            name: Messages.REPLUGGED_PLUGIN,
+            displayName: `List plugins`, // TODO: i18n
+            value: "plugin",
+          },
+        ],
+      },
+      {
+        name: "version",
+        displayName: Messages.REPLUGGED_COMMAND_LIST_OPTION_VERSION_NAME,
+        description: Messages.REPLUGGED_COMMAND_LIST_OPTION_VERSION_DESC,
+        type: ApplicationCommandOptionType.Boolean,
+        required: false,
+      },
+      {
+        name: "status",
+        displayName: Messages.REPLUGGED_COMMAND_LIST_OPTION_STATUS_NAME,
+        description: Messages.REPLUGGED_COMMAND_LIST_OPTION_STATUS_DESC,
+        type: ApplicationCommandOptionType.String,
+        required: false,
+        choices: [
+          {
+            name: Messages.REPLUGGED_COMMAND_LIST_OPTION_STATUS_CHOICE_ENABLED,
+            displayName: Messages.REPLUGGED_COMMAND_LIST_OPTION_STATUS_CHOICE_ENABLED,
+            value: "enabled",
+          },
+          {
+            name: Messages.REPLUGGED_COMMAND_LIST_OPTION_STATUS_CHOICE_DISABLED,
+            displayName: Messages.REPLUGGED_COMMAND_LIST_OPTION_STATUS_CHOICE_DISABLED,
+            value: "disabled",
+          },
+          {
+            name: Messages.REPLUGGED_COMMAND_LIST_OPTION_STATUS_CHOICE_BOTH,
+            displayName: Messages.REPLUGGED_COMMAND_LIST_OPTION_STATUS_CHOICE_BOTH,
+            value: "default",
+          },
```

Comment:
`name` and `displayName` seems redundant

Reply:
both are needed as far as i remember and i dont wanna make another loop inside the loop for options to map these with displayName too.